src/osyris/spatial/subdomain.py

```diff
@@ -28,9 +28,9 @@ def extract_sphere(dataset, radius, origin):
     return subdomain
 
 
-def extract_box(dataset, dx, dy, dz, origin):
+def extract_box(dataset, xmin, xmax, ymin, ymax, zmin, zmax, origin):
```

Comment:
Coming back to this, I wasn't necessarily saying we should remove it, maybe we can keep both? e.g. `extract_box` would be the one with dx, `extract_range` could be with xmin/xmax?

Reply:
I don't really like the name `extract_range`. Can we think of something else?
Also, could we not have a single function that can do both?